src/components/templates/UserProfileForm.jsx

```diff
@@ -160,7 +160,7 @@ const UserProfileForm = ({initialState}) => {
                           Cancel
                         </Button>
                       </Link>
-                      <Button color="green" type="submit">
+                      <Button color="green" to="/dashboard/profile" type="submit">
```

Comment:
Why add the `to` on the submit button @mariehposa

Reply:
That was a mistake and could be found on other submit buttons too.

Reply:
I didn't work on the user profile form, only the user profile.